tfhe/docs/getting_started/_operations.md

```diff
@@ -0,0 +1,64 @@
+# Supported Operations
```

Comment:
what's the purpose of this _operations.md file ? It seems it's not linked at the moment, is it supposed to be replaced or is it the newer version of the operations.md file ?